run/test/specs/disappearing_messages.spec.ts

```diff
@@ -60,13 +63,251 @@ async function disappearingMessages(platform: SupportedPlatformsType) {
+    strategy: "accessibility id",
+    selector: "Control message",
+    text: `You have set messages to disappear 10 seconds after they have been sent`,
+  });
+  // Check control message is correct on device 2
+  await device2.findConfigurationMessage(
+    `${userA.userName} has set messages to disappear 10 seconds after they have been sent`
+  );
+  // Send message to verify that deletion is working
+  await device1.sendMessage(testMessage);
+  await device2.clickOnElementByText({
+    strategy: "accessibility id",
+    selector: "Message body",
+    text: testMessage,
+  });
+  // Wait for 10 seconds
+  await sleepFor(10000);
+  // Check if message is deleted in device 1
+  await device1.hasTextElementBeenDeleted("Message body", testMessage);
+  // Check on device 2
+  await device1.hasTextElementBeenDeleted("Message body", testMessage);
+  // Great success
+  await closeApp(device1, device2);
+}
+
+async function disappearAfterRead(platform: SupportedPlatformsType) {
+  const { device1, device2 } = await openAppTwoDevices(platform);
+  // Create user A and user B
+  const [userA, userB] = await Promise.all([
+    newUser(device1, "Alice", platform),
+    newUser(device2, "Bob", platform),
+  ]);
+  const testMessage = "Checking disappear after read is working";
+  // Create contact
+  await newContact(platform, device1, userA, device2, userB);
+  // Click conversation options menu (three dots)
+  await device1.clickOnElement("More options");
+  // Select disappearing messages option
+  await device1.clickOnElement("Disappearing messages");
+  await device1.clickOnElement("Disappear after read option");
+  // Need to validate that default time is checked somehow
+  await device1.waitForTextElementToBePresent({
+    strategy: "accessibility id",
+    selector: "12 hours",
+  });
+  await device1.radioButtonSelected("12 hours");
+  // Change timer to ten seconds (testing time)
+  await device1.clickOnElement("10 seconds");
+  // Click on set to save setting
+  await device1.clickOnElement("Set button");
+  await device1.navigateBack(platform);
+  await sleepFor(1000);
+  await device1.waitForTextElementToBePresent({
+    strategy: "accessibility id",
+    selector: "Control message",
+    text: `You have set messages to disappear 10 seconds after they have been read`,
+  });
+  // Check control message is correct on device 2
+  await device2.findConfigurationMessage(
+    `${userA.userName} has set messages to disappear 10 seconds after they have been read`
+  );
+
+  // Send message to verify that deletion is working
+  await device1.sendMessage(testMessage);
+  // Wait for 10 seconds
+  await sleepFor(10000);
+  // Check if message is deleted in device 1
+  await device1.hasElementBeenDeletedNew({
+    strategy: "accessibility id",
+    selector: "Message body",
+    text: testMessage,
+    maxWait: 1000,
+  });
+  // await sleepFor(9000);
+  // Check on device 2
+  await device2.hasElementBeenDeletedNew({
```

Comment:
do you still have a  non new functionl ike this one? If not, you can refactor it to `hasBeenDeleted`